pkg/rpcclient/wsclient.go

```diff
@@ -127,7 +127,7 @@ func (r *blockReceiver) EventID() neorpc.EventID {
 }
 
 // Filter implements neorpc.Comparator interface.
-func (r *blockReceiver) Filter() any {
+func (r *blockReceiver) Filter() neorpc.SubscriptionFilter {
```

Comment:
Change the implementation accordingly. E.g. `func (r *headerOfAddedBlockReceiver) Filter() neorpc.SubscriptionFilter {` and ditto for the others implementations. Somehow I thought you had this code before, and now it's gone. The linter is unhappy about it and the project can't be built.